incubator/binding-grpc.spec/src/main/resources/META-INF/zilla/grpc.idl

```diff
@@ -14,5 +14,53 @@
  */
 scope grpc
 {
-	
+    scope stream
+    {
+        enum GrpcKind
+        {
+            UNARY,
+            STREAM
+        }
+
+        enum GrpcType
+        {
+            TEXT,
+            BASE64
+        }
+
+        struct GrpcMetadata
+        {
+            GrpcType type;
+            varint32 nameLen;
+            octets[nameLen] name;
+            varint32 valueLen;
+            octets[valueLen] value = null;
+        }
+
+        struct GrpcBeginEx extends core::stream::Extension
+        {
+            string16 scheme;
+            string16 authority;
```

Comment:
Do we also need `path` to be able to proxy non-standard paths between `grpc` client and `grpc` server?

Reply:
What I am doing on the client side is concatenating service and method fields to construct the path.